ui/logviewer.html

```diff
@@ -97,6 +97,8 @@
                   <a ng-cloak ng-switch-when="link" title="{{line.value}}"
                      href="{{line.url}}" target="_blank">{{line.value}}</a>
                     <span ng-switch-when="raw_html" ng-bind-html="line.value"></span>
+                    <span ng-switch-when="object">{{line.value}}</span>
```

Comment:
Hm. I can't see the point of this if we default to the same value. I'm going to remove it.